tests/integration/test_properties.py

```diff
@@ -26,19 +26,7 @@ def test_post_property(client, auth_headers, new_property):
 def test_get_property_by_name(client, auth_headers, test_database):
     """The get property by name returns a successful response code."""
     response = client.get("/api/properties/test1", headers=auth_headers["admin"])
-    property_info = response.get_json()
     assert response.status_code == 200
-    assert property_info['name'] == 'test1'
-    assert property_info['address'] == '123 NE FLanders St'
-    assert property_info['unit'] == '5'
-    assert property_info['city'] == 'Portland'
-    assert property_info['state'] == 'OR'
-    assert property_info['zipcode'] == '97207'
-    assert property_info['propertyManager'] == 5
-    assert property_info['propertyManagerName'] == 'Gray Pouponn'
-    assert property_info['tenantIDs'] == [1]
-    assert property_info['dateAdded'] == '2020-04-12'
-    assert property_info['archived'] == 0
```

Comment:
what's the reason for deleting this?

Reply:
I believe the assert statements are more like unit test like you mentioned in the lease model. We should use the serializer to test the json return value once in unit test later.

Reply:
yes, I agree. However, I think we should at least test that it returns a json response of the requested resource, as that is its main responsibility. And eventually we should unit test the json response. Otherwise these tests do hold some value at this point, because I could replace 'return rentalProperty.json()' with 'return PropertyModel.query.first().json()' and it passes..